windows/RNCNetInfoCPP/RNCNetInfo.cpp

```diff
@@ -162,15 +174,26 @@ namespace winrt::ReactNativeNetInfo::implementation {
                             details.frequency = network.ChannelCenterFrequencyInKilohertz() / 1000; // Convert to Mhz
                             details.wifiGeneration = GetWifiGeneration(network.PhyKind());
                         }
+                        if (signal) {
+                            details.strength = winrt::unbox_value<uint8_t>(signal) * 20; // Signal strength is 0-5 but we want 0-100.

```

Comment:
Why was signal strength removed from cell connections? Does it not work for cell?

Reply:
I was sticking to the details provided in the README. I've reverted it since we are now showing all the properties